samples/grpc/shopping-analytics-service-java/pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <akka-grpc-maven-plugin.version>2.2.0</akka-grpc-maven-plugin.version>
+        <akka-grpc-maven-plugin.version>2.2.1</akka-grpc-maven-plugin.version>
```

Comment:
also update the akka-dependencies to 22.10.3 ?

Reply:
Akka-dependencies was updated to 22.10.3, but testing failing now. Probably flakiness. 

I restarted them.